src/Func.cpp

```diff
@@ -863,6 +865,103 @@ void Func::invalidate_cache() {
     }
 }
 
+Func Func::in(const Func &f) {
+    user_assert(name() != f.name()) << "Cannot call 'in()' on itself\n";
+    const map<string, IntrusivePtr<FunctionContents>> &wrappers = func.wrappers();
+    const auto &iter = wrappers.find(f.name());
+    if (iter == wrappers.end()) {
+        string wrapper_name = name() + "_in_" + f.name();
+        wrapper_name = replace_all(wrapper_name, "$", "");
```

Comment:
I don't think you should need this replace_all line any more, the recently change to unique_name should make it unnecessary.